examples/step-9/step-9.cc

```diff
@@ -465,7 +400,7 @@ namespace Step9
   // risk that the write operations from two cells might conflict, and the
```

Comment:
In line 396:  only ONE thread gets to do this at a time.

Reply:
Fixed.